src/Libraries/Microsoft.Extensions.AI.Evaluation.Reporting/TypeScript/azure-devops-report/src/azure-devops-extension-api/Common/Client.ts

```diff
@@ -0,0 +1,37 @@
+import { getAccessToken, getService } from "azure-devops-extension-sdk";
```

Comment:
Applies to all files in this PR: Please add header comment (same as other code files in the repo).